docs/ext/thread-pools.md

```diff
@@ -0,0 +1,65 @@
+# Thread Pools
+
+Java's [ThreadPoolExecutor](http://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ThreadPoolExecutor.html)
+exposes several properties that are useful to monitor to assess the health, performance, and configuration of the pool.
+
+## Getting Started
+
+To report thread pool metrics, one can attach a
+[ThreadPoolMonitor](http://netflix.github.io/spectator/en/latest/javadoc/spectator-api/com/netflix/spectator/api/patterns/ThreadPoolMonitor.html)
+in the following manner:
+
+```java
+import com.netflix.spectator.api.patterns.ThreadPoolMonitor;
+import com.netflix.spectator.api.Spectator;
+
+ThreadPoolMonitor.attach(Spectator.registry(), myThreadPoolExecutor, "my-thread-pool");
+
```

Comment:
Remove empty line

Reply:
Weird... in the diff it shows an empty line, but the actual file doesn't have one.